header.html

```diff
@@ -1,6 +1,6 @@
 <div>
     <a target=_blank href="http://openmobilealliance.org/"><img src="/Images/oma_logo.jpg"></a>
-    <a target=_blank href="http://openmobilealliance.org/feed/"><img src="/Images/oma_Feed.jpg " HEIGHT="40" WIDTH="40" BORDER="0"></a>
+    <a target=_blank href="http://openmobilealliance.org/feed/"><img src="/Images/oma_Feed.JPG " HEIGHT="40" WIDTH="40" BORDER="0"></a>

```

Comment:
File extensions are case insensitive. So it doesn't matter if it was upper case or not. Usually filename extensions are kept at the same case as the file name.